### PR TITLE
Handle failure for SGE ACH

### PR DESCRIPTION
**Pull request by dfuller114**
Confirm that ACH errors display consistently with credit card errors in single gift entry. Also, we are disabling the Elevate widget whenever we receive an error from a successful purchase call submission so that the user cannot change the payment method and re-render the widget once a payment has already been sent to Elevate.

Post PR in Future Features doc for release notes.

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by dfuller114** [comment]
This PR is for final code review. #6164 was the original PR but was closed due to an error in the branch name

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -6708,8 +6708,8 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <categories>Gift Entry</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Failure description if card is charged and donation record has not been created</shortDescription>
-        <value>The card has been successfully charged but the payment has not been recorded in Salesforce. Please correct the errors and resubmit the form to record the donation in Salesforce.</value>
+        <shortDescription>Failure msg when transaction is processed but record isn't created in Salesforce</shortDescription>
+        <value>Elevate successfully processed the payment, but NPSP didn't create the record for the transaction in Salesforce. Please correct the errors and resubmit the form.</value>
```
Comment (melissabarber) [inline src/labels/CustomLabels.labels:6712]:
```suggestion
        <value>Elevate successfully processed the payment, but NPSP didn't create the record for the transaction in Salesforce. Please correct the errors and resubmit the form to create the Salesforce record.</value>
```

Comment (dfuller114) [reply src/labels/CustomLabels.labels]: @melissabarber updated!

**Review by ptewson-sfdo — COMMENTED** [review]
This looks good Daniel! My only comment is about method naming (related convo-starter in  our dev channel). If that doesn't resolve in time to close this out, I would NOT consider that comment blocking this story. However, updating the name of the function should not change behavior.

**Review by ptewson-sfdo — APPROVED** [review]
Looks good!

**Review by psadie-sfdo — APPROVED** [review]
Looks good based on our conversations and expectations!

**Comment by dfuller114** [comment]
@psadie-sfdo @ptewson-sfdo I added a few commits to fix some additional issues Deepa came across that were downstream effects so wanted to get one more code review on these

**Pull request merged**